libs/shared/ebpf_serialize.c

```diff
@@ -477,6 +481,7 @@ ebpf_serialize_program_info(
             for (uint16_t index = 0; index < EBPF_COUNT_OF(helper_prototype->arguments); index++) {
                 serialized_helper_prototype->arguments[index] = helper_prototype->arguments[index];
             }
+            serialized_helper_prototype->flags.reallocate_packet = helper_prototype->flags.reallocate_packet;
```

Comment:
```suggestion
            serialized_helper_prototype->flags = helper_prototype->flags;
```
Copying the entire flags struct.  (Works fine if you typedef the flags struct)
This way you don't have to update this code if more flags are added.